sycl/source/detail/os_util.cpp

```diff
@@ -93,19 +113,27 @@ size_t OSUtil::getOSMemSize() {
   MemInfo.dwLength = sizeof(MemInfo);
   GlobalMemoryStatusEx(&MemInfo);
   return static_cast<size_t>(MemInfo.ullTotalPhys);
-#endif
+#elif defined(SYCL_RT_OS_DARWIN)
+  int64_t Size;
+  sysctlbyname("hw.memsize", &Size, nullptr, nullptr, 0);
+  return static_cast<size_t>(Size);
+#endif // SYCL_RT_OS
 }
 
 void *OSUtil::alignedAlloc(size_t Alignment, size_t NumBytes) {
 #if defined(SYCL_RT_OS_LINUX)
   return aligned_alloc(Alignment, NumBytes);
+#elif defined(SYCL_RT_OS_POSIX_SUPPORT)
+  void *Addr;
+  int ReturnCode = posix_memalign(&Addr, Alignment, NumBytes);
+  return (ReturnCode==0) ? Addr : nullptr;
```

Comment:
Is this in accordance with clang-format?

Reply:
Fixed by 4c52bee2521aa42c9ed993e98e3d5c311f8678c5